articles/native-platforms/ionic/00-intro.md

```diff
@@ -3,8 +3,28 @@ title: Introduction
 description: test
 ---
 
+<%= include('../../_includes/_package', {
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '00-Starter-Seed',
+  pkgFilePath: '00-Starter-Seed/www/app/auth0.variables.js',
+  pkgType: 'replace'
+}) %>
+
 Welcome to this quickstart which will guide you through the various steps of implementing Auth0 in your Ionic applications.
 
+<%= include('../../_includes/_package', {
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples/tree/master/00-Starter-Seed',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '00-Starter-Seed',
+  pkgFilePath: '00-Starter-Seed/www/app/auth0.variables.js',
+  pkgType: 'replace'
+}) %>
+
 Auth0 provides and manages an [AngularJS SDK](https://github.com/auth0/auth0-angular) which can be used with Ionic applications. This SDK wraps Auth0's [Lock Widget](https://github.com/auth0/lock) as well as the [Auth0.js library](https://github.com/auth0/auth0.js), so it makes it simple to integrate these in your Ionic applications. 
```

Comment:
@jerriep Can you remove please one of the includes? https://auth0-docs-content-pr-1963.herokuapp.com/docs/quickstart/native/ionic